pom.xml

```diff
@@ -21,12 +21,17 @@
         <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-core</artifactId>
-            <version>6.2.7.Final</version>
+            <version>6.5.2.Final</version>
         </dependency>
         <dependency>
-            <groupId>mysql</groupId>
-            <artifactId>mysql-connector-java</artifactId>
-            <version>8.0.33</version>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-c3p0</artifactId>
```

Comment:
why do you need this dependency?

Reply:
If I don't use this dependency I get the following warning: "WARN: HHH10001002: Using built-in connection pool (not intended for production use)".